### DISCUSSION TITLE
TARGET BAR SIZE value is bigger than VRAM SIZE

### DISCUSSION THREAD

**Discussion by MirageNesher** [terminatorul/NvStrapsReBar#84] [Q&A]
GTX 1660 6GB
I followed the instructions, flashed the BIOS, launched NvStrapsReBar and used the E and S keys for auto-setting, and everything went well, re-bar was activated for all 6/6GB. But after some time, about a week of use, I come across that the re-bar is disabled and the application displays code 0x32. I reset the parameters for the GPU and repeated the auto-setting, but now for some reason the target vram size is 8GB instead of 6. In manual configuration mode it is also impossible to set 6GB, there are parameters only for 4 or 8GB. Is it safe to use, is everything as it should be?
![изображение](https://github.com/terminatorul/NvStrapsReBar/assets/151023142/55f3807c-5590-48ed-805c-c3233f612789)
![изображение](https://github.com/terminatorul/NvStrapsReBar/assets/151023142/2fc2febf-e374-4050-88c3-499dc33a5708)
![изображение](https://github.com/terminatorul/NvStrapsReBar/assets/151023142/30439b8f-bfb8-4a3d-acd3-60befb64241d)

**Comment by terminatorul** [comment]
The target BAR size is correct. The BAR size must be a power of two. Having a BAR size of 8 GBytes with 6 GBytes of VRAM is expected and is the recommended size to use. There is no way to set a BAR size of 6 GBytes, the hardware does not support it.
